setup.py

```diff
@@ -32,60 +32,59 @@
 ]
 
 TESTS_REQUIRE = [
-    'pytest >= 3.6.3',
-    'pytest-cov',
-    'coverage',
-    'flake8',
-    'nbval==0.9.6',  # https://github.com/computationalmodelling/nbval/issues/194
-    'pytest-mock',
+    "pytest >= 3.6.3",
+    "pytest-cov",
+    "coverage",
+    "flake8",
+    # nbval greater than 0.9.6 has a bug with semicolon
+    # https://github.com/computationalmodelling/nbval/issues/194
+    "nbval<=0.9.6",
+    "pytest-mock",
 ]
 
 INSTALL_REQUIRES = [
-    'matplotlib >= 3.0.0',
-    'numpy >= 1.17.3, <2.0',
-    # pandas restricted to <2.1 until
-    # https://github.com/pandas-dev/pandas/issues/55794
-    # is resolved
-    'pandas >= 1.3.0, <2.1',
-    'statsmodels >= 0.11.1',
-    'scipy >= 1.2.0',
-    'h5py >= 2.8.0',
-    'plotly>=4.0.0',
-    'xgboost >= 1.3.3',
-    'pvlib >= 0.7.0, <0.12.0',
-    'scikit-learn >= 0.22.0',
-    'arch >= 4.11',
-    'filterpy >= 1.4.2'
+    "matplotlib >= 3.5.3",
+    "numpy >= 1.22.4",
+    "pandas >= 1.4.4",
+    "statsmodels >= 0.13.5",
+    "scipy >= 1.8.1",
+    "h5py >= 3.7.0",
+    "plotly>=4.0.0",
+    "xgboost >= 1.3.3",
+    "pvlib >= 0.9.0, <0.12.0",
```

Comment:
requirments_min specifies 0.11. We should change the minimum here too right?

Reply:
Absolutely right, I overlooked that. I've bumped it to 0.11 now.